src/Playground.vue

```diff
@@ -1,74 +1,105 @@
+        <l-polyline
+          :lat-lngs="[
+            [47.334852, -1.509485],
+            [47.342596, -1.328731],
+            [47.241487, -1.190568],
+            [47.234787, -1.358337],
+          ]"
+          color="green"
+        ></l-polyline>
+        <l-polygon
+          :lat-lngs="[
+            [46.334852, -1.509485],
+            [46.342596, -1.328731],
+            [46.241487, -1.190568],
+            [46.234787, -1.358337],
+          ]"
+          color="#41b782"
+          :fill="true"
+          :fillOpacity="0.5"
+          fillColor="#41b782"
+        />
+        <l-rectangle
+          :lat-lngs="[
+            [46.334852, -1.509485],
+            [46.342596, -1.328731],
+            [46.241487, -1.190568],
+            [46.234787, -1.358337],
+          ]"
+          :fill="true"
+          color="#35495d"
+        />
+        <l-rectangle
+          :bounds="[
+            [46.334852, -1.190568],
+            [46.241487, -1.090357],
+          ]"
+        >
+          <l-popup>
+            lol
+          </l-popup>
+        </l-rectangle>
 
-      <l-polyline
-        :lat-lngs="[
-          [47.334852, -1.509485],
-          [47.342596, -1.328731],
-          [47.241487, -1.190568],
-          [47.234787, -1.358337],
-        ]"
-        color="green"
-      ></l-polyline>
-      <l-polygon
-        :lat-lngs="[
-          [46.334852, -1.509485],
-          [46.342596, -1.328731],
-          [46.241487, -1.190568],
-          [46.234787, -1.358337],
-        ]"
-        color="#41b782"
-        :fill="true"
-        :fillOpacity="0.5"
-        fillColor="#41b782"
-      />
-      <l-rectangle
-        :lat-lngs="[
-          [46.334852, -1.509485],
-          [46.342596, -1.328731],
-          [46.241487, -1.190568],
-          [46.234787, -1.358337],
-        ]"
-        :fill="true"
-        color="#35495d"
-      />
-      <l-rectangle
-        :bounds="[
-          [46.334852, -1.190568],
-          [46.241487, -1.090357],
-        ]"
+        <l-polyline
+          :lat-lngs="[
+            [47.334852, -1.509485],
+            [47.342596, -1.328731],
+            [47.241487, -1.190568],
+            [47.234787, -1.358337],
+          ]"
+          color="green"
+        ></l-polyline>
+      </l-map>
+      <button @click="changeIcon">New kitten icon</button>
+    </div>
+    <div style="height: 75vh; width: 50vw;">
+      <l-map
+        v-model="zoom"
+        v-model:zoom="zoom"
+        :center="[47.41322, -1.219482]"
+        @move="log('move')"
       >
-        <l-popup>
-          lol
-        </l-popup>
-      </l-rectangle>
-    </l-map>
-    <button @click="changeIcon">New kitten icon</button>
+        <l-wms-tile-layer
+          v-for="layer in wmsLayers"
+          :key="layer.name"
+          :base-url="baseUrl"
+          :layers="layer.layers"
+          :visible="layer.visible"
+          :name="layer.name"
+          layer-type="base"
+          >></l-wms-tile-layer
+        >
```

Comment:
```suggestion
        ></l-wms-tile-layer>
```
Minor formatting and not having an extra `>` as the layer contents